src/main/java/com/customer/service/CreditCardService.java

```diff
@@ -9,4 +9,8 @@
     public CreditCard createCreditCardInfo(CreditCard creditCardInfo);
 
     List<CreditCard> getAllCreditCardsInfo(int year);
+
+    public void deleteCreditCardInfoById(int id);
+
+    public void deleteAllCreditCardInfo();
```

Comment:
why creditCard INFO ?

renaming everything to creditCard

Reply:
Done.